lifemonitor/commands/db.py

```diff
@@ -94,3 +99,87 @@ def wait_for_db():
     while current_revision is None:
         current_revision = db_revision()
     logger.info(f"Current revision: {current_revision}")
+
+
+@cli.db.command()
+@click.option("-f", "--file", default=None, help="Backup filename (default 'hhmmss_yyyymmdd.tar')")
+@click.option("-d", "--directory", default="./", help="Directory path for the backup file (default '.')")
+@with_appcontext
+def backup(file, directory):
+    """
+    Make a backup of the current app database
+    """
+    from lifemonitor.db import db_connection_params
+    params = db_connection_params()
+    if not file:
+        file = f"{datetime.now().strftime('%Y%m%d_%H%M%S')}.tar"
+    target_path = os.path.join(directory, file)
+    cmd = f"PGPASSWORD={params['password']} pg_dump -h {params['host']} -U {params['user']} -F t {params['dbname']} > {target_path}"
+    os.system(cmd)
```

Comment:
The code is not checking the exit code from the called process.  Rather than `os.system`, we'd be better off just using `subprocess.check_call`.

Reply:
addressed in https://github.com/crs4/life_monitor/pull/210/commits/3a72fa2e633938d929053fa94f8df26a9cf9958c